src/menu/components/boxart.c

```diff
@@ -51,6 +51,33 @@ component_boxart_t *component_boxart_init (const char *storage_prefix, char *gam
     return NULL;
 }
 
+component_boxart_t *component_boxart_init_dd (const char *storage_prefix, char *game_code) {
```

Comment:
Do we need another function for that? I'd rather reuse existing function with some changes regarding maximum supported image resolution.